<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plugins::ConnectionManager::ConnectionWidget</class>
 <widget class="QWidget" name="Plugins::ConnectionManager::ConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>482</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cmbConnectionType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnConnect">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../OpenSpeedShop/Resources/OpenSpeedShop.qrc">
         <normaloff>:/OpenSpeedShop/connect.png</normaloff>:/OpenSpeedShop/connect.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnDisconnect">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../OpenSpeedShop/Resources/OpenSpeedShop.qrc">
         <normaloff>:/OpenSpeedShop/disconnect.png</normaloff>:/OpenSpeedShop/disconnect.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QStackedWidget" name="stackedWidget"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../OpenSpeedShop/Resources/OpenSpeedShop.qrc"/>
 </resources>
 <connections/>
</ui>
